lib/galaxy/tool_util/verify/interactor.py

```diff
@@ -591,13 +601,13 @@ def _element_identifiers(self, collection_def):
             element_identifiers.append(element)
         return element_identifiers
 
-    def __dictify_output_collections(self, submit_response):
+    def __dictify_output_collections(self, submit_response) -> Dict[str, Any]:
         output_collections_dict = {}
         for output_collection in submit_response["output_collections"]:
             output_collections_dict[output_collection.get("output_name")] = output_collection
```

Comment:
I suppose with don't want `output_collections_dict[None]`.

```suggestion
            output_collections_dict[output_collection["output_name"]] = output_collection
```

Similarly at line 617 there's `outputs_dict[output.get("output_name")] = output` that I think should be `outputs_dict[output["output_name"]] = output`